_saas-integrations/recharge/v1/recharge-v1.md

```diff
@@ -0,0 +1,133 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: ReCharge
+permalink: /integrations/saas/recharge
+keywords: recharge, integration, schema, etl recharge, recharge etl, recharge schema
+layout: singer
+# input: false
+
+key: "recharge-setup"
+
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "recharge"
+display_name: "ReCharge"
+
+singer: true
+status-url: ""
+
+tap-name: "recharge"
+repo-url: https://github.com/singer-io/tap-recharge
+
+this-version: "1.0"
+
+api: |
+  [ReCharge Payments API](https://developer.rechargepayments.com/){:target="new"}
+
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+certified: true 
+
+historical: "1 year"
+frequency: "1 hour"
+tier: "Free"
+
+anchor-scheduling: true
+cron-scheduling: true
+
+extraction-logs: true
+loading-reports: true
+
+table-selection: true
+column-selection: true
+
+# attribution-window: "# days"
+# attribution-is-configurable: 
+
+# setup-name: ""
+
+
+# -------------------------- #
+#      Feature Summary       #
+# -------------------------- #
+
+feature-summary: |
+  Stitch's {{ integration.display_name }} integration replicates data using the {{ integration.api | flatify | strip }}. To access the {{ integration.display_name }} API documentation, you will need [your API key](#obtain-api-key). Refer to the [Schema](#schema) section for a list of objects available for replication.
+
+
+
+# -------------------------- #
+#      Incompatibilities     #
+# -------------------------- #
+
+## uncomment section below if integration is compatible with any Stitch destinations
+## if incompatible with multiple destinations, create a section for each destination
+
+## incompatible:
+  ## [redshift]: "always,sometimes,never"
+  ## reason: "copy" 
+
+# -------------------------- #
+#      Setup Instructions    #
+# -------------------------- #
+
+requirements-list:
+  - item: "**A Shopify Account**. You need this to access your {{ integration.display_name }} information."
+
+setup-steps:
+  - title: "Obtain Your {{ integration.display_name }} API Key"
+    anchor: "obtain-api-key"
+    content: |
+      1. Log into your Shopify account, select `Apps`, and then click on your {{ integration.display_name }} application.
```

Comment:
For button, field, and page/section names, let's use **bold** text instead of `code`. This is consistent with the rest of our docs at the moment. We tend to use `code` to indicate actual values for something - ex: "In the **Name** field, enter `Stitch`", `Stitch` being the thing we actually want to enter into the **Name** field.